src/api/sessionApi.js

```diff
@@ -12,10 +12,22 @@ class Session {
   static logout() {
     return api.delete('/users/sign_out');
   }
-
   static signUp(user) {
     return api.post('/users', user);
   }
+  static updateUser(user) {
+    const { id } = user;
+    return api.put(`users/${id}`, { user });
+  }
+  static getTopics() {
+    return api.get('topics');
+  }
+  static newTarget(target) {
```

Comment:
Let's make a standard to name these kind of functions starting with a verb. So `createTarget` maybe?